pkg/utils/utils.go

```diff
@@ -70,3 +71,17 @@ func DecompressGZIP(compressedData []byte) (decompressedData []byte, err error)
 
 	return decompressedData, nil
 }
+
+func HumanReadableSize(data []byte) string {
+	const unit = 1024
+	size := len(data)
+	if size < unit {
+		return fmt.Sprintf("%d B", size)
+	}
+	div, exp := int64(unit), 0
+	for n := size / unit; n >= unit; n /= unit {
+		div *= unit
+		exp++
+	}
+	return fmt.Sprintf("%.1f %cB", float64(size)/float64(div), "KMGTPE"[exp])
```

Comment:
Given that you are doing 2^10 rather than 10^3 (disk manufacturer) multipliers, I would use the unambiguous format, i.e. KiB, MiB, etc.

Also probably better to keep 3 places of precision given the multiplier stepping you are using.

```suggestion
	return fmt.Sprintf("%.3f %ciB", float64(size)/float64(div), "KMGTPE"[exp])
```